frontend/packages/knative-plugin/src/components/overview/RevisionsOverviewList.tsx

```diff
@@ -46,18 +44,7 @@ const RevisionsOverviewListItem: React.FC<RevisionsOverviewListItemProps> = ({
 
 const RevisionsOverviewList: React.FC<RevisionsOverviewListProps> = ({ revisions, service }) => (
   <>
-    <SidebarSectionHeading text="Revisions" className="revision-overview-list">
-      {/* add extra check, if sidebar is opened for a knative deployment */}
-      {service.kind === ServiceModel.kind && (
-        <Button
-          variant="secondry"
-          onClick={() => trafficModalLauncher({ obj: service })}
-          disabled={!(revisions && revisions.length)}
-        >
-          Set Traffic Distribution
-        </Button>
-      )}
-    </SidebarSectionHeading>
```

Comment:
@karthikjeeyar Do we not need this on the sidebar resource tab? There is a Pr to fix the styling of `Set Traffic Distribution` button. https://github.com/openshift/console/pull/3349.

Reply:
cc @vikram-raj

Reply:
The issue description says ```Set Traffic Distribution``` should be part of the action and context menu as well.

Reply:
@vikram-raj @sahil143 Added the button in the sidebar again, I thought the button is temporary until we get this action into context-menu and sidebar action.